.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Feature Request
+about: Suggest a new idea/feature for RepoSense
+labels: "c.Feature"
+---
+
+<!--
+  Before opening a new issue, please search existing issues:  https://github.com/reposense/RepoSense/issues
+-->
+
+**Is your request related to a problem?**
+
+<!--
+  Provide a clear and concise description of what the problem is.
+  Ex. I have an issue when [...]
+-->
```

Comment:
I feel a user who has thought of a feature he wants, would also have a solution in his mind regarding what he wants. It might be a good idea, to ask them their idea. 

Can make it optional while motivating them to suggest their idea.

Reply:
Addressed